bigtable-hbase-dataflow/README.md

```diff
@@ -0,0 +1,3 @@
+# Bigtable Dataflow
```

Comment:
This is a big PR....

Bigtable HBase Dataflow? To match the path.

On that note... This uses HBase, mention it in the documentation? Will there be a non-HBase dataflow?
Does this belong to cloud-bigtable-client repo? Why not keep it in a separate repo, since it is justa a use of the cbt client and not part of it? Also, was this developed in a separate repo? should we try to keep the commits?

Also is it worth creating integration tests? Specially since it interacts with another system.